tutor_signup.html

```diff
@@ -32,41 +32,73 @@ <h2 class="logo"> <a href="./index.html">UBtutoring</a> </h2>
     <h1 class="welcome-page-title">Sign Up</h1>
 
     <div id="tutor_signup_div">
-        <form method="post" action="">
-            <label for="fname">First Name</label>
+        <form>
+            <label>(Fields with asterisks '*' must be filled and valid in order to create an account)</label>
+            <br>
+            <label for="fname">First Name *</label>
 
-            <input class="sign_up_input" type="text" id="fname" name="firstname" placeholder="Your name..">
+            <input class="sign_up_input" type="text" id="fname" name="firstname" placeholder="Your first name.">
 
-            <label for="lname">Last Name</label>
-            <input class="sign_up_input" type="text" id="lname" name="lastname" placeholder="Your last name..">
+            <label for="lname">Last Name *</label>
+            <input class="sign_up_input" type="text" id="lname" name="lastname" placeholder="Your last name.">
             
-            <label for="email">Buffalo Email</label>
-            <input class="sign_up_input" type="text" id="email" name="email" placeholder="Enter @buffalo.edu email..">
+            <label for="email">Buffalo Email *</label>
+            <input class="sign_up_input" type="text" id="email" name="Email" placeholder="Enter @buffalo.edu email.">
```

Comment:
I'd stay away from changing form variables unless you explicitly know why / have to. Most of the php / js functions depend on these variable names and if you change them without understanding why, it will cause things to break. revert back to email.